.github/scripts/diff-directories.js

```diff
@@ -0,0 +1,159 @@
+const fs = require('fs')
+const path = require('path')
+const diff = require('diff')
+
+function readFilesRecursively (directory) {
+    const filenames = fs.readdirSync(directory)
+    const files = {}
```

Comment:
Any fears that if the config or platforms grow that this object will become unmanageably large?

Reply:
I wouldn't say so, this should manage 1000s of files without it being a performance boon I would say. (we could always cut off the legacy ones if becomes an issue too)

Were there other concerns about number of files?

Reply:
Mostly just memory related concerns. We'll see if it becomes an issue later on ✨